vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package quality (aka metrics_quality) - code for VPA quality metrics
+package quality
+
+import (
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/utils/metrics"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	metricsNamespace = metrics.TopMetricsNamespace + "quality"
+)
+
+var (
+	usageRecommendationRelativeDiff = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
+			Name:      "usage_recommendation_relative_diffs",
+			Help:      "Diffs between usage and recommendation, normalized by recommendation value",
+			Buckets: []float64{-1., -.75, -.5, -.25, -.1, -.05, -0.025, -.01, -.005, -0.0025 - .001,
```

Comment:
-0.0025 - .001, - something wrong happened here
And we lack 0, I think.

Reply:
Indeed. Fixed. Thanks for the attentive review :)